hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyRenameRequest.java

```diff
@@ -118,7 +119,8 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
             OMException.ResultCodes.INVALID_KEY_NAME);
       }
       // check Acl
-      checkKeyAcls(ozoneManager, volumeName, bucketName, fromKeyName);
+      checkKeyAcls(ozoneManager, volumeName, bucketName, toKeyName,
+          IAccessAuthorizer.ACLType.CREATE);
```

Comment:
It is arguable to only check CREATE for the toKeyName as we are remove/delete the fromKey as well.

Reply:
Here do we need to check whether the originalKey acls also?

Reply:
@bharatviswa504 You mean DELETE acl on fromKeyName? I think it might be a good idea to also check for DELETE acl on fromKeyName

@xiaoyuyao What do you think?

Reply:
Discussed offline, we should have DELETE check for fromKeyName and CREATE check for toKeyName.